src/app/hierarchical-grid/hierarchical-grid-selection/hierarchical-grid-selection.component.ts

```diff
@@ -34,11 +37,20 @@ export class HGridSelectionSampleComponent implements OnInit, OnDestroy {
 
     public selectCellSelectionMode(args) {
         this.selectionMode = this.selectionModes[args.index].label;
+        this.snackbarRowCount.close();
         this.snackbar.open();
+        this.selectedRowsCount = undefined;
+        this.selectedRowIndex = undefined;
     }
 
     public handleRowSelection(event) {
```

Comment:
@IvanKitanov17 you've decided not to type the `rowSelection` event this time with `IRowSelectionEventArgs`?

Reply:
@zdrawku  It is re-added with the latest commit.